mirage/psf/soss_trace.py

```diff
@@ -514,13 +538,13 @@ def SOSS_psf_cube(filt='CLEAR', order=1, subarray='SUBSTRIP256', generate=False,
         trace_cols = np.arange(2048)
 
         # Run datacube
-        for n, wavelength in enumerate(wavelengths):
+        for n, wavelength in enumerate(wavelengths[:3, :]):
 
             # Evaluate the trace polynomial in each column to get the y-position of the trace center
             trace_centers = np.polyval(coeffs[n], trace_cols)
 
             # Don't calculate order2 for F277W or order 3 for either
```

Comment:
This is inconsistent with the line below. Should just say no order 2 or 3 for F277W?